src/components/Gallery/ImagePagination.tsx

```diff
@@ -47,7 +47,10 @@ export function ImagePagination({
             indicateDropdown
             onClick={() => setOpen(!isOpen)}
           >
-            {'Page ' + pageInfo.current + ' of ' + pageInfo.totalPageCount}
+            {'Page ' +
+              `${pageInfo.currentIndex + 1}` +
+              ' of ' +
+              pageInfo.totalPageCount}
```

Comment:
```suggestion
            {`Page ${pageInfo.currentIndex + 1} of ${pageInfo.totalPageCount}`}
```

Reply:
Unfortunately, this is enforced by the linter 🙁

Reply:
good catch 😅